src/submission/NodeSubmissionClient.ts

```diff
@@ -16,44 +16,35 @@ export class NodeSubmissionClient extends DefaultSubmissionClient {
     this.configurationVersionHeader = this.configurationVersionHeader.toLowerCase();
   }
 
-  public sendRequest(config:Configuration, method:string, path:string, data:string, callback: (status:number, message:string, data?:string, headers?:Object) => void): void {
-    function complete(response:http.IncomingMessage, responseBody:string, responseHeaders:Object) {
-      var message:string;
-      if (response.statusCode === 0) {
-        message = 'Unable to connect to server.';
-      } else if (response.statusCode < 200 || response.statusCode > 299) {
-        message = response.statusMessage || (<any>response).message;
-      }
+  public sendRequest(config: Configuration, method: string, path: string, data: string, callback: NodeSubmissionCallback): void {
```

Comment:
This was part of refactoring out the actual "send via http" code, which I wanted to get really concise to generate a small submitSync.js that depends on it. 

I think we should make this non-node-specific as soon as we start to support the same behavior in the browser, e.g. by using the experimental SendBeacon feature that enqueues a request in the unload-handler. This could work similarly.

Though, I wanted to keep the footprint of the exceptionless.js as small as possible, therefore not doing unnecessary changes yet.